webknossos-tracingstore/app/com/scalableminds/webknossos/tracingstore/tracings/FossilDBClient.scala

```diff
@@ -78,39 +79,41 @@ class FossilDBClient(collection: String,
   }
 
   private def wrapException[T](future: Future[T]): Fox[T] =
-    future.transformWith {
-      case Success(value) =>
-        Fox.successful(value).futureBox
-      case Failure(exception) =>
-        val box = exception match {
-          case e: StatusRuntimeException if e.getStatus == Status.UNAVAILABLE =>
-            new net.liftweb.common.Failure(s"FossilDB is unavailable", Full(e), Empty) ~> 500
-          case e: Exception =>
-            val messageWithCauses = new StringBuilder
-            messageWithCauses.append(e.toString)
-            var cause: Throwable = e.getCause
-            while (cause != null) {
-              messageWithCauses.append(" <- ")
-              messageWithCauses.append(cause.toString)
-              cause = cause.getCause
-            }
-            new net.liftweb.common.Failure(s"Request to FossilDB failed: $messageWithCauses", Full(e), Empty)
-        }
-        Future.successful(box)
+    Fox.fromFutureBox {
+      future.transformWith {
+        case Success(value) =>
+          Fox.successful(value).futureBox
```

Comment:
Very nitpicky and this wasn't part of your pr but:

Hmm here you construct a fox to get the future box and then construct a fox again. Below a future box is constructed instead.

So maybe something like `Future.successful(Full(value))` might be more consistent with the `Future.successful(box)` below